tests/unit/univariate/test_t.py

```diff
@@ -0,0 +1,46 @@
+from unittest import TestCase
+
+import numpy as np
+from scipy.stats import t
+
+from copulas.univariate.student_t import StudentTUnivariate
+
+
+class TestStudentTUnivariate(TestCase):
+
+    def test__fit_constant(self):
+        distribution = StudentTUnivariate()
+
+        distribution._fit_constant(np.array([1, 1, 1, 1]))
+
+        assert distribution._params == {
+            'df': 100,
+            'loc': 1,
+            'scale': 0
+        }
+
+    def test__fit(self):
+        distribution = StudentTUnivariate()
+
+        data = t.rvs(size=1000, df=3, loc=1, scale=1)
+        distribution._fit(data)
+
+        assert distribution._params == {
+            'df': t.fit(data)[0],
```

Comment:
Let's not rely on third party computations on the checks.
The expected values should be specified as literal values and validated using numpy's `assert_all_close`.

You can use other univariate unit tests, like  `test_gamma.py` or `test_beta.py`, as reference.